includes/Classifai/Command/ClassifaiCommand.php

```diff
@@ -279,6 +350,49 @@ private function get_posts_to_classify( $opts = [] ) {
 		return $posts;
 	}
 
+	/**
+	 * Returns the list of attachment ids to classify with Azure Compute Vision
+	 *
+	 * @param array $opts Options from WP CLI.
+	 * @return array
+	 */
+	private function get_attachment_to_classify( $opts = [] ) {
+		$limit = is_numeric( $opts['limit'] ) ? $opts['limit'] : - 1;
```

Comment:
Let's not use -1 here, if there are 100,000 images on a site where this is being run it could be very bad. I would recommend setting an upper limit here.

If we're looking to batch all items, we should split this out into separate requests using pagination but keep the `posts_per_page` lower number.

Many of the tips here https://10up.github.io/Engineering-Best-Practices/migrations/#tips-to-speed-up-migrations might be nice additions to this as we're not guaranteed that this plugin will be run on a server that can handle processing a very large amount of data.